docs/maps/maps-getting-started.asciidoc

```diff
@@ -62,14 +61,15 @@ and lighter shades will symbolize countries with less traffic.
 
 . Add a Tooltip field:
 
-** Select **ISO 3166-1 alpha-2 code** and **name**.
-** Click **Add**.
+** **ISO 3166-1 alpha-2 code** is added by default.
+** Click **Add** and select **name**.
+** Click **Add** to create the field.
 
 . In **Layer style**, set:
 
 ** **Fill color: As number** to the grey color ramp
 ** **Border color** to white
-** **Label** to symbol label
+** **Label** to Fixed - symbol label
```

Comment:
Can you change to 
```
** Under **Label**, change *By value* to *Fixed*.
```

Reply:
I suggest making this parallel:

In **Layer style**:

**  Set **Fill color: As number** to the grey color ramp.
**  Set **Border color** to white.
**   Under **Label**, change **By value** to **Fixed**.